client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -35,7 +35,7 @@ import { noRetry } from 'state/data-layer/wpcom-http/pipeline/retry-on-failure/p
 
 import { registerHandlers } from 'state/data-layer/handler-registry';
 
-const changeCommentStatus = ( { dispatch, getState }, action ) => {
+const changeCommentStatus = action => ( dispatch, getState ) => {
```

Comment:
Note that after we fix #27154, we can further simplify this handler. The `previousStatus` will be a property of the `COMMENTS_CHANGE_STATUS` action and we won't have to do any work getting it from `getState` and storing it to `meta.comment.previousStatus`.